### PR TITLE
copy stdin to clipboard from integrated terminal

### LINKED ISSUE #181724: Clipboard support from integrated terminal
## What is your suggestion?

On linux (and WSL2), I like to use clipboard utilities to paste the stdout of a process to the clipboard.

Like the following:

```
date | xclip -selection clipboard 
```

Currently, it is not possible to do something similar if developing remotely.

It would be great if the vscode would have the functionality `code --clipboard` to allow this.

So that the following would lead to the text to be in the clipboard:

```
echo -n my clipboard text |  code --clipboard
```

## Why do you want this feature?

When interacting with tokens etc. it would be a nice feature to hide the tokens from any output and history.

## Are there any workarounds to get this functionality today?

Select the output of the application, press CTRL+SHIFT+C.

But that is cumbersome and also in case of passwords, prints the passwords to the terminal which has to be cleared afterwards.

### PR DESCRIPTION
**Pull request by smerschjohann**
Fixes #181724

to test use a remote-cli.sh script pointing to the dev environment and run it from an integrated terminal.

e.g.:

```bash
#!/usr/bin/env sh
#
# Copyright (c) Microsoft Corporation. All rights reserved.
#
VSROOT=/home/coder/dev/vscode
ROOT="$(dirname "$(dirname "$VSROOT")")"


APP_NAME="code"
VERSION="1.0.0"
COMMIT="..."
EXEC_NAME="code"
CLI_SCRIPT="$VSROOT/out/server-cli.js"
"${NODE_EXEC_PATH:-$ROOT/lib/node}" "$CLI_SCRIPT" "$APP_NAME" "$VERSION" "$COMMIT" "$EXEC_NAME" "$@"
```

usage in terminal:

```bash
echo -n my clipboard text | code --clipboard
```

### CODE REVIEW

**Comment by smerschjohann** [comment]
@microsoft-github-policy-service agree

File: src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ class RemoteTerminalBackend extends BaseTerminalBackend implements ITerminalBack
 			}
 		});
 
-		const allowedCommands = ['_remoteCLI.openExternal', '_remoteCLI.windowOpen', '_remoteCLI.getSystemStatus', '_remoteCLI.manageExtensions'];
```
Comment (Tasha471) [inline src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts:91]: const allowedCommands = ['_remoteCLI.openExternal', '_remoteCLI.windowOpen', '_remoteCLI.getSystemStatus', '_remoteCLI.manageExtensions', '_remoteCLI.setClipboard'];

Comment (smerschjohann) [reply src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts:91]: I cannot see a difference in your comment to what I wrote. Can you describe what you mean?

**Comment by smerschjohann** [comment]
With the new API: https://code.visualstudio.com/updates/v1_82#_listen-to-terminal-command-execution it see the potential of implementing it as an extension.

**Pull request closed without merging**